src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -168,11 +168,11 @@ public void ProcessStart_UseShellExecute_OnWindows_DoesNotThrow(bool isFolder)
             {
                 if (isFolder)
                 {
-                    Assert.Null(px); // Not sure why px returned is null. but the call does not throw and opens folder successfully.
+                    Assert.Null(px);
                 }
                 else
                 {
-                    if (px != null) // sometimes process is null: tracked by #24048
```

Comment:
I would leave the comments (since the information is useful)_ but just remove `: tracked by #24048`

Comments are good, bug numbers can always be dug up from commit messages.